airbyte_cdk/sources/declarative/manifest_declarative_source.py

```
@@ -238,6 +238,11 @@ def _migrate_and_transform_config(
         self._spec_component.transform_config(mutable_config)
         return mutable_config
 
+    def configure(self, config: Mapping[str, Any], temp_dir: str
+    ) -> Mapping[str, Any]:
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)